recipe/migrations/azure_storage_files_datalake_cpp12110.yaml

```
@@ -0,0 +1,8 @@
+__migrator:
+  build_number: 1
+  commit_message: Rebuild for azure_storage_files_datalake_cpp 12.11.0
+  kind: version
+  migration_number: 1
+azure_storage_files_datalake_cpp:
+- 12.11.0
```

```suggestion
- 12.11.0
azure_storage_queues_cpp:
- 12.3.0
```

Let's combine with https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6209